examples/getting-started-typescript/src/app.ts

```diff
@@ -2,7 +2,6 @@
 /* eslint-disable import/no-internal-modules */
 import './utils/env';
 import { App, LogLevel } from '@slack/bolt';
-import { isGenericMessageEvent } from './utils/helpers';
```

Comment:
Removed the utility class to make the example code available for cutting and pasting.